site/profile/manifests/archive/common.pp

```diff
@@ -32,14 +32,29 @@
     ensure_packages($packages)
   }
 
-  class { 'python':
-    version     => 'python36',
-    pip         => 'present',
-    dev         => 'present',
-    venv        => 'present',
-    python_pips => $python_pips,
+  if fact('os.architecture') == 'x86_64' {
```

Comment:
@glennmorris As of https://github.com/lsst-it/puppet-ccs_software/pull/31 , I assume the alternative link is now undesirable? Do you want me to make a new release of `lsst/ccs_software` and bump the version in this PR?

Reply:
@jhoblitt Yes, I would like a new release of ccs_software please. IMO the version bump could precede this PR, but having it in here is fine too.